frontstage/controllers/party_controller.py

```diff
@@ -229,32 +230,116 @@ def get_respondent_enrolments(party_id):
                 }
 
 
+def set_enrolment_data(enrolment_data):
+    surveys_ids = set()
+    business_ids = set()
+    for enrolment in enrolment_data:
```

Comment:
Maybe needs a comment here to explain that we're figuring out the unique set of surveys and parties, so that we don't get the data more than once

Reply:
Sounds good 👍